.github/workflows/ci.yml

```diff
@@ -63,8 +63,6 @@ jobs:
       # The commit being checked out is the merge commit for the PR. Its first
       # parent will be the tip of main.
       BASE_REF: HEAD^
-      PR_TITLE: ${{ github.event.pull_request.title }}
-      PR_BODY: ${{ github.event.pull_request.body }}
```

Comment:
I'm not convinced we want this (I commented on https://github.com/openxla/iree/issues/10042).

As much as possible we should aim to have our workflows follow the standard practices that GitHub Actions encourages. This is changing the behavior of "re-running a workflow" (https://docs.github.com/en/actions/managing-workflow-runs/re-running-workflows-and-jobs).

Reply:
I'm still not satisfied with this setup... but I can get behind the overall priorities. My specific concern with the behavior here is: re-running [failed] workflows should do what devs expect. If we add our own gating logic and caching, that could do unexpected things or be difficult to debug.

Reply:
Yeah this is why I had it this way in the first place. I actually think that devs *don't* expect that rerunning will work this way. People have multiple times complained that they did a rerun after changing the description and it didn't change anything.

Regarding failed workflows: "rerun failed" won't rerun the setup job, so the output will not change. Only if you "rerun all" will setup rerun. I don't really see any use in rerunning all unless you *expect* a change in behavior.

I hear your concerns and very much share them, but this approach is after much wrestling and experimenting with GitHub actions in an attempt to let us do something reasonable with it. Alternative suggestions are welcome though. You really have to look into all the details though: many things do not function as you might expect and missing functionality abounds.